### PR TITLE
Oss 8

### PR DESCRIPTION
**Pull request by AsifBhar**
- Please let me know where do we need to deploy this maven artifact.
- Give me api key which we can be saved in yaml file so that jar is uploaded in git hub repo.
- if we plan to deploy artifact to repo.systemkern.com. Please share the creds so that we can upload over there.
- let me know if I am missing something.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  
+before_deploy:
+  - mvn help:evaluate -N -Dexpression=project.version|grep -v '\['
+  - export project_version=$(mvn help:evaluate -N -Dexpression=project.version|grep -v '\[')
+
+script:
+# -B batch mode makes Maven less verbose
+  - mvn -B clean verify
+
+before_deploy:
```
Comment (rainerkern) [inline .travis.yml]: i think the before_deploy section is duplicated

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    - mvn help:evaluate -N -Dexpression=project.version|grep -v '\['
+    - export project_version=$(mvn help:evaluate -N -Dexpression=project.version|grep -v '\[')
+
+## Create release in GitHub    
+deploy:
+    provider: releases
+    api_key:
+        secure: kEmbwstjQDAjsl3TR4d0rMFP99ZL4t9Sa19Bfx2FfwlStiwWyLDw9bmNGkYQyuvpMTh6igWfvx1sTvOhKqVGDCdj89/5GJquIrfPNJSJXPr3KP4zdU6xEhK57gGCa4IfMF5jcAe24z6fOrPLzEVyqMBrIxu8RQryqfqPtVvRj+uDI0xn6IPxGZ6nc97IRcIykwufNChsMOavE595FXn9WmVSBd/O+VY4LLefRLOFWxL1dTGlUxJLzSAYEVTCxYnz+lbCmfbyreu9eSmDZRnti78cjss2cHDzkInl/P1U7AXj6My0JJ8+TsVHt/HBLYDsEotv/ayTxH859qjDpvkL3FRoXEv4PtVn8TsyUjfJoCwwU5/pvFGElIlE2l6tpfnmSKkJ70eADIzox4Qi+biNx7M+mXC8tl7S0r+mSI2kjJgUMAZjjbbZQnBQ+RQCPKcseB0X2bXKxOmFb+t4OGZ9BXZWmxFXcljgiSu+Fbz2VRaTPG3OSjjIiOPeE3NdfJwoHrqBDcMzsh0c+UCTjpqghI0jHPZe8BcySEjJ5yUM74cUtGuhPru4Hs0xTKOKUubUFQ9oOsKLSgWNg28Px6BhMbEucs/8AZHtT1EiGEJPV7JB/i7qVPYGhGhWKF4id08GOhh+pd1lbNyO+Y0G1wG+VBtRjiqPIIN+NqXZbfi4goM=
```
Comment (rainerkern) [inline .travis.yml]: yeah … could you please not put api keys into publicly available files … i hope i do not have to explain why this is a bad idea

also: where does this api key come from?

Comment (AsifBhar) [reply .travis.yml]: yes @rainerkern I totally understand it.  Actually  this key is used to upload the binary to our github repo. Will remove it.

File: .travis.yml (hunk 1 above)
Comment (AsifBhar) [reply .travis.yml]: yes :(

**Comment by rainerkern** [comment]
Hi Asif, so how exactly is this deployment going to work?

**Comment by AsifBhar** [comment]
Hello Rainer,

For FTP Upload I think we can upload jar to FTP location :-https://docs.travis-ci.com/user/deployment/custom/

 For Maven central I think we can do it by :-https://www.phillip-kruger.com/post/continuous_integration_to_maven_central/

Let me know if I am missing something. 

Regards,
Asif

**Pull request closed without merging**